tools/ci_build/github/windows/setup_env_eager.bat

```diff
@@ -0,0 +1,2 @@
+set PATH=C:\azcopy;C:\\hostedtoolcache\\windows\\Python\\3.7.9\\x64\\lib\\site-packages\\torch\\lib;%PATH%
```

Comment:
Generally speaking, PATH env is used for locating EXEs not DLLs. We never add site-packages dir to PATH. So I suspect something was wrong.